### ISSUE TITLE
 6-2: Softmax classifier 의 cost함수

### ISSUE THREAD

**Issue by NahahaNahaha** [NahahaNahaha/tensorflow#13]
<img width="450" alt="2018-04-07 11 08 01" src="https://user-images.githubusercontent.com/26895961/38455844-92d1dbec-3ab8-11e8-9704-05e0f1e8b15d.png">
여기서 D는 지난번 cost 함수의 C(H(x),y) = D(S,L) 는 같다고 보면 된다. 
그 이유는 H(x) 는 log(Si) 와 같고 실제 값과 다른 경우 무한대로 발산하고 같은 경우 0으로 수렴하기 때문이다.
